docs/stencil-docs/installing-stencil-cli/live-previewing-a-theme.md

```diff
@@ -40,18 +40,18 @@ To develop against BigCommerce's Cornerstone theme (which is the building block
 git clone https://github.com/bigcommerce/cornerstone.git
 ```
 
-Cornerstone and other themes can also be downloaded from the BigCommerce control panel. For instructions on doing so, see [Downloading and Uploading Custom Themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload) (BigCommerce Knowledge Base).
+Cornerstone and other themes can also be downloaded from the BigCommerce control panel. For instructions on doing so, see [Downloading and Uploading Custom Themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload) (BigCommerce Knowledge Base). 
 
 It is also possible to download and develop against themes in the BigCommerce Themes Marketplace. To do so:
 1. Navigate to **Storefront** > **Theme Marketplace**, and add the desired theme.
 2. Once added, navigate to **Storefront** > **My Themes** > **Theme Thumbnail** > **Theme Options**.
 3. Select the appropriate Download option.
-
-
+ 
 **Download options**:
 * **Download current theme** - download the theme active on the storefront (appears when current theme is selected).
-* **Download your latest customizations** - download the theme's most recently saved version (appears only for themes customized for this store).

```

Comment:
why are we removing this one ? Also I wonder if we even need this `Download Options` section for cli operations